src/Microsoft.Extensions.CodeGenerators.Mvc/Controller/ControllerGen.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Extensions.CodeGeneration;
+using Microsoft.Extensions.CodeGeneration.CommandLine;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.Extensions.CodeGenerators.Mvc.Controller
+{
+    [Alias("controller")]
+    public class ControllerGenerator : ICodeGenerator
```

Comment:
:-), yeah, I wasn't very sure if git gets confused because another file with the name as this class was renamed in my local repo ... I corrected the file name in a later check in...